src/analyserbeats.cpp

```diff
@@ -33,14 +33,16 @@ AnalyserBeats::AnalyserBeats(ConfigObject<ConfigValue>* pConfig)
 AnalyserBeats::~AnalyserBeats() {
 }
 
-bool AnalyserBeats::initialise(TrackPointer tio, int sampleRate, int totalSamples) {
+bool AnalyserBeats::initialise(TrackPointer tio, int sampleRate,
+                               int totalSamples) {
     if (totalSamples == 0) {
         return false;
     }
 
     bool bPreferencesBeatDetectionEnabled = static_cast<bool>(
-        m_pConfig->getValueString(
-            ConfigKey(BPM_CONFIG_KEY, BPM_DETECTION_ENABLED)).toInt());
+            m_pConfig->getValueString(
+                             ConfigKey(BPM_CONFIG_KEY, BPM_DETECTION_ENABLED))
+                    .toInt());
```

Comment:
Do we mind how much more room this takes up? I can see the value of having it split out though, like here, it's now hard to miss the .toInt().
(What do the rules do if the ConfigKey() bit would have extended past the defined line length?)